src/main/java/appeng/core/sync/AppEngPacket.java

```diff
@@ -19,60 +19,52 @@
 package appeng.core.sync;
 
 
-import io.netty.buffer.ByteBuf;
+import cpw.mods.fml.common.network.simpleimpl.IMessage;
 
-import net.minecraft.entity.player.EntityPlayer;
 
-import cpw.mods.fml.common.network.internal.FMLProxyPacket;
-
-import appeng.core.AEConfig;
-import appeng.core.AELog;
-import appeng.core.features.AEFeature;
-import appeng.core.sync.network.INetworkInfo;
-import appeng.core.sync.network.NetworkHandler;
-
-
-public abstract class AppEngPacket
+public interface AppEngPacket extends IMessage
```

Comment:
The later ones are as return packets. The packethandlers allow to directly return a new packet as answer for the received one. I simply have not looked, if there are any predefined ones provided by forge, which could be reused.

But currently we never create an answer, thus it is really not needed but required by forge. So I just left it at `IMessage` and not a more specific `AppEngPacket`.